sidekiq/README.md

```diff
@@ -0,0 +1,14 @@
+# Sidekiq
+
+- Prefer to place workers in the `/app/workers` folder
+- Prefer to follow the [Sidekiq Best Practices](https://github.com/mperham/sidekiq/wiki/Best-Practices)
+- Use `raise` on the `perform` method to tell Sidekiq that a job has failed when using `rescue` or `Result`
+- Prefer to log the `error` message and `backtrace` on rescue, to help debugging
```

Comment:
O sidekiq ja loga ambos quando um erro é retornado

Reply:
@clsechi sua sugestão é remover essa linha `- Prefer to log the ``error`` message and ``backtrace`` on rescue, to help debugging`?

Reply:
dando uma olhada no worker, se a gente não faz nenhum tratamento customizado no `rescue`, podemos remover ele por completo e deixar o sidekiq logar os erros

Mas é boa prática logar os erros em recues, mas isso cabe a outra section

Reply:
Boa! Eu acho que poderíamos logar mesmo assim, acaba ficando redundante já que vai pra dois lugares, mas se alguém começar a investigar algo pelo papertrail por exemplo, já conseguiria ver também sem ir para o painel do sidekiq. O que acha de manter o raise + loggar o erro mesmo?

Reply:
O sidekiq loga com o `Rails.logger.warn`, então vai td pro papertrail, a unica diferença é não vai com a log_tag, ai é um ponto a se discutir

Reply:
Na backguilda falamos que não precisa loggar, vamos tirar essa linha aí.